src/Benchmarks.ClientJob/Latency.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Benchmarks.ClientJob
+{
+    public class Latency
+    {
+        public double Average { get; set; }
```

Comment:
I would use `TimeSpan` unless there's a good reason to use `double`.

Reply:
I was averaging values, and had to convert to ms before. I can return a TimeSpan back, ok.